solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -108,6 +115,11 @@ contract RandomBeacon is Ownable {
     ///         `maliciousDkgResultSlashingAmount`.
     uint256 public maliciousDkgResultSlashingAmount;
 
+    /// @notice This amount is required to protect against griefing the staking
+    ///         contract and individual applications are allowed to require higher
+    ///         minimum stakes if necessary.
+    uint96 public minimumStake;
```

Comment:
Not sure if I follow. Why do we need to store it here? We can read it from the sortition pool. If we need to keep it here, how this param will be set?

Reply:
Ok, thanks for the explanation!